### ISSUE TITLE
Is this app still maintained?

### ISSUE THREAD

**Issue by osoitz** [RomanGolovanov/ametro#148]
The version in F-Droid seems not to work any more.
Is this app still maintained?

https://gitlab.com/fdroid/fdroiddata/issues/1893

**Comment by Chewie9999** [comment]
I don't think so. ametro.org just goes to some crappy news site, and I can't download any new maps :(
Shame.
